contrib/configsnap/README.md

```diff
@@ -0,0 +1,55 @@
+# configsnap
+
+Run configsnap at each major phase of auter. Configsnap records useful system state information and creates backups of some configuration files. These can be compared at different points in time based on tags.
+
+# Script Details
+
+Language: bash
+
+Supported OS: CENTOS>=6 RHEL>=6 Fedora>=26 Ubuntu>=16.04 Debian>=9
```

Comment:
It might be worth removing the Debian based systems as we don't currently have a package available for installation on those distros?